CLITest.yml

```diff
@@ -0,0 +1,316 @@
+# Some content of this file is generated.
+
+name: CLI TEST RUN $(USER_TARGET) $(USER_LIVE) $(Date:yyyyMMdd)$(Rev:.r)
```

Comment:
As far as I understand, thoses files are not related to user usage. So, it shoud be placed inside at the root level of project.

Reply:
I'll move it to live test folder

Reply:
Done